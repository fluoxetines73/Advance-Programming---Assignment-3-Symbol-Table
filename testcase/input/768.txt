INSERT d number
RPRINT
BEGIN
LOOKUP d
BEGIN
BEGIN
LOOKUP d
BEGIN
ASSIGN d 123.5
INSERT f_FF string
INSERT y string
PRINT
RPRINT
LOOKUP f_FF
PRINT
RPRINT
ASSIGN f_FF 'UPPERlower'
ASSIGN d 'UPPERlower'
INSERT e_123_2r number
BEGIN random
INSERT e_123_2r string
LOOKUP f_FF
END random
PRINT
END
ASSIGN d 456 random
END
END
END