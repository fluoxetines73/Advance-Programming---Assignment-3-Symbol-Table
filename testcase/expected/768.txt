Invalid: ASSIGN d 123.5